src/pages/hosts/HostsTable.tsx

```diff
@@ -117,7 +117,7 @@ export const HostsTable: React.VFC<Props> = ({
       sorter: true,
       className: "cy-hosts-table-col-ID",
       defaultSortOrder: getDefaultSortOrder(HostSortBy.Id),
-      width: "15%",
+      width: "17%",
```

Comment:
nit.  This comment is more so about my personal feelings toward "magic numbers" as opposed to an actual complaint.  How would this look if we used more rounded numbers? Like maybe 15% and 20% (on line 171)